journal/modernized/06-chapter-5.adoc

```diff
@@ -752,18 +752,7 @@ had recently undergone a change.
 He hoped to be able to live so as to be united to the true disciples of Christ,
 or words to this import.
 We left this person in a very tender frame of mind,
-and parted under solemn feelings.gospel of our Lord and Savior Jesus Christ;
-and I had large and open service among them, greatly to the relief of my mind.
-The prospect of this meeting had for months past borne with weight on my mind,
-and now I was enabled through holy help to cast off a great burden.
-Blessed, praised and magnified be the name of Israel`'s Shepherd.
-He brings low and raises up, and is worthy of all glory and honor forever.
-I was largely opened in this meeting to declare against infidelity, atheism,
-and all unrighteousness; and had to bear my testimony against those works of darkness,
-called spiritual rappings, and the workers thereof; stating my belief,
-that it was the devil and his agents who carried it on, and were the originators thereof.
-A young man, a stranger to us,
-expressed his satisfaction with what he had heard delivered.
+and parted under solemn feelings.
```

Comment:
This big chunk that I erased didn't belong there (bottom of page 111). Belongs to another paragraph on top of the p. 113 (and it's actually already there!)